Add scroll buttons to tabs when they go beyond the available space (#4538)
marmelab/react-admin: docs/CreateEdit.md

* [`margin`](#margin)
* `save`: The function invoked when the form is submitted. This is passed automatically by `react-admin` when the form component is used inside `Create` and `Edit` components.
* `saving`: A boolean indicating whether a save operation is ongoing. This is passed automatically by `react-admin` when the form component is used inside `Create` and `Edit` components.

> * `scrollable`: A boolean, `true` by default. `<TabbedForm>` use MaterialUI scrollable buttons when there is too much FormTabs to display. If you preferer the screen to expands with the tabs, set it `false`.

Reviewer: ```suggestion
* `scrollable`: A boolean, `true` by default. `<TabbedForm>` use MaterialUI scrollable buttons when there are too many FormTabs to display. If you prefer the screen to expands with the tabs, set it `false`.
```
